.github/workflows/main.yml

```diff
@@ -6,10 +6,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: [ 2.5, 2.6, 2.7, jruby ]
+        ruby: [jruby, 2.5, 2.6, 2.7]
```

Comment:
No problem with changing the order here, but was this on purpose and if so what is the reasoning behind the order?

Reply:
This was on purpose; I find it easier to think about since the latest Ruby version is also the last item in the list and the one that we run linters/benchmarks on.